astropy/coordinates/baseframe.py

```diff
@@ -1366,6 +1366,58 @@ def sphericalcoslat(self):
         return self.represent_as('spherical', 'sphericalcoslat',
                                  in_frame_units=True)
 
+    @property
+    def velocity(self):
+        """
+        Shorthand for the Cartesian velocity as a `CartesianDifferential`
+        object.
```

Comment:
Perhaps also note that it is equivalent to ``self.cartesian.differentials['s']``?  I think this is a likely route users will take to starting to understand the necessary complexity of the differential stuff.

And maybe also "velocity" -> "space motion"? but that's not a big deal either way.